frontend/src/app/app-routing.module.ts

```diff
@@ -10,7 +10,7 @@ const routes: Routes = [
   {path: 'header', component: HeaderComponent},
   {path: 'footer', component: FooterComponent},
   {path: 'callback', component: CallbackComponent},
-  {path: '**', component: HomeComponent},
+  {path: '**', component: HomeComponent}
```

Comment:
Not sure how Angular really works, but just want to be sure that this extremely non-specific glob is supposed to be here.